app/controllers/questions_controller.rb

```diff
@@ -120,6 +120,11 @@ def destroy
       redirect_to(question_path(@question)) && return
     end
 
+    if @question.answer_count > 0 then
+      flash[:danger] = 'Error 1729'
```

Comment:
Error messages should be useful - let's make this say something like "This question cannot be deleted because it has answers."

(There is an exception for errors for trolls, but those are defined elsewhere)

Reply:
Would claim that us both are to co-blame for the message choice.

Reply:
Eh, there's no blame necessary - we just need this one to say something useful :)